book/_toc.yml

```diff
@@ -34,6 +34,10 @@
           sections:
             - file: tutorials/sar/sentinel1
             - file: tutorials/sar/uavsar
+        - file: tutorials/thermal-ir/index
+          sections:
+            - file: tutorials/thermal-ir/thermal-ir-1
```

Comment:
Looks like you need to add `thermal-ir-0.ipynb` here (which downloads the necessary file for subsequent notebooks. How large are these files? We could also put them here as a backup https://github.com/snowex-hackweek/tutorial-data

Reply:
Updated to use a single notebook for the main tutorial with datasets downloaded from s3. I have extra notebooks and .py files that I didn't include in the toc with the idea that that those are accessible through other hyperlinks or when the repo is cloned to jupyterhub.